pt-br/csharp-pt.html.markdown

```diff
@@ -784,7 +784,7 @@ on a new line! ""Wow!"", the masses cried";
 
     } // end class Bicycle
 
-    // PennyFarthing is a subclass of Bicycle
+    // PennyFarthing é uma subclasse de Bicycle
     class PennyFarthing : Bicycle
     {
         // (Penny Farthings are those bicycles with the big front wheel.
```

Comment:
Could you please translate this comment, also?

Reply:
Ok!